packages/Uploader/src/components/DropZone/DropZone.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import tokens from "@paprika/tokens";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import UploadIcon from "@paprika/icon/lib/Upload";
+import { UploaderContext } from "../../Uploader";
+import * as div from "./DropZone.styles";
```

Comment:
The convention we seem to have settled on was to import styled components like this as `sc` (vs `div`).